runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -2322,6 +2322,20 @@ J9::SymbolReferenceTable::findOrCreateProfilingBufferEndSymbolRef()
    return element(profilingBufferEndSymbol);
    }
 
+TR::SymbolReference *
+J9::SymbolReferenceTable::findOrCreateTempSlotSymbolRef()
```

Comment:
This is going to need a different name - in the compiler a temp slot is a stack allocated variable - this usage will be very confusing. I would suggest findOrCreateVMThreadTempSlotSymbolRef or something explicit like that.

Reply:
Please add doxygen for what this is

Reply:
Addressed your suggestions.